app/src/crash/crash-app.tsx

```diff
@@ -91,22 +91,43 @@ export class CrashApp extends React.Component<ICrashAppProps, ICrashAppState> {
     super(props)
 
     this.state = {
-      windowState: getWindowState(remote.getCurrentWindow()),
+      windowState: null,
     }
+
+    this.initializeWindowState()
   }
 
   public componentDidMount() {
-    const window = remote.getCurrentWindow()
-
-    ipcRenderer.on('window-state-changed', () => {
-      this.setState({ windowState: getWindowState(window) })
-    })
+    ipcRenderer.on('window-state-changed', this.onWindowStateChanged)
```

Comment:
Not sure why this wasn't using the window-state sent by `window-state-changed` event. Switched it to follow same logic in our `windows-controls.tsx` for the main app.